cmd/crossbuild.go

```diff
@@ -136,8 +148,12 @@ func runCrossbuild() {
 	}
 
 	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" {
```

Comment:
if the first letter would be lowercase, it's fine to keep all lowercase. Just don't use `Mips`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms